airflow/providers/http/hooks/http.py

```diff
@@ -246,3 +253,142 @@ def test_connection(self):
             return True, "Connection successfully tested"
         except Exception as e:
             return False, str(e)
+
+
+class HttpAsyncHook(BaseHook):
+    """
+    Interact with HTTP servers using Python Async.
+
+    :param method: the API method to be called
+    :param http_conn_id: http connection id that has the base
+        API url i.e https://www.google.com/ and optional authentication credentials. Default
+        headers can also be specified in the Extra field in json format.
+    :param auth_type: The auth type for the service
+    """
+
+    conn_name_attr = "http_conn_id"
+    default_conn_name = "http_default"
+    conn_type = "http"
+    hook_name = "HTTP"
```

Comment:
I'm not sure that this required.
This mostly for test connection and new connection in the UI but we already register conn_type = "http" for HttpHook

Reply:
I think it's in parity with `HttpHook` https://github.com/apache/airflow/blob/main/airflow/providers/http/hooks/http.py#L50